src/pages/IDResolutionPage/IDResolveRedirectionPage.tsx

```diff
@@ -14,12 +12,15 @@ import {
 } from './IDResolvedManyPage';
 
 const IDResolveRedirectionPage = () => {
-  const nexus = useNexusContext();
   const navigate = useHistory().push;
   const { apiEndpoint, basePath } = useSelector((state: RootState) => ({
     apiEndpoint: state.config.apiEndpoint,
     basePath: state.config.basePath,
   }));
+  const [{ error, isError }, setResolutionState] = useState({
```

Comment:
Two things about this state:

1. I think the name `setResolutionError` or ``setResourceResolutionError` is more suited here because we only store error information in this state, nothing else
2. Where are we using the value of `error` state? Should we perhaps add a notification displaying this error?

Reply:
1. 👍 
2. the error will be showed in the modal